### ISSUE TITLE
make jbang edit http... possible

### ISSUE THREAD

**Issue by maxandersen** [jbangdev/jbang#580]
it would be nice if we could just to `jbang edit gavsearch@jbangdev` and then have it in case of a git backed file just git clone
the project and then use https://github.com/jbangdev/jbang/issues/462 to edit the directory letting users easily contribute/fork a script/app.

might want something more fine grained than a single `edit` command ...requires some thinking what would be best (incl. not doing it but have it as a "plugin")

**Comment by knutwannheden** [comment]
Or possibly something like `jbang catalog clone jbangdev`?

**Comment by quintesse** [comment]
I think that using that to clone any github/gitlab/etc repository would overload the meaning of "catalog" a bit too much. Catalogs are repos containing scripts, not just any repos. At least IMO.

**Comment by knutwannheden** [comment]
Yes, the idea would be to have an easy way to clone the catalog repo, so that a script of that repo can be modified.
